.github/workflows/build-arch-emu.yaml

```diff
@@ -0,0 +1,87 @@
+name: arch-emu
+
+# this is done on workflow_dispatch only since it takes a very long time
+on:
+  workflow_dispatch:
+
+jobs:
+
+  alpine:
+    # Run armv7 on aarch64 runners
+    runs-on: ${{ matrix.arch == 'armv7' && 'ubuntu-24.04-arm' || 'ubuntu-latest' }}
+
+    defaults:
+      run:
+        # Use emulated shell as default
+        shell: alpine.sh {0}
+
+    strategy:
+      matrix:
+        config:
+          # FIXME: switch to 10.1.0 when it is released:
```

Comment:
Before we release a stable v1.2, I will have a GraphBLAS v10.1.0 ready.  That version of GraphBLAS has an important feature:  the ability to force GraphBLAS.h to not expose any GxB methods to the user application, with

#define GRAPHBLAS_VANILLA
#include <GraphBLAS.h>

That addition to GraphBLAS enabled me to find some GxB's that had snuck into the LAGraph "vanilla" build.